exercises/random_number.cpp

```diff
@@ -4,3 +4,15 @@
   Output:
   The random number is: 4
 */
+
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+int main(){
+  srand(time(0));
+
+  cout << "The random number is: " << rand() << endl;
```

Comment:
C'è da aggiungere, alla fine del **main**, il 
```c
return 0;
```

Oppure, ancora meglio, si potrebbe aggiungere in cima al file
```c
#include <iostream>
#define EXIT_SUCCESS 0

using namespace std
```
e aggiungere alla fine del **main**

```c
return EXIT_SUCCESS;
```